doc/source/user/tutorial-svd.rst

```diff
@@ -122,146 +95,107 @@ can access this by using the syntax
            [ 85,  95, 112, ..., 121, 120, 120],
            [ 85,  97, 111, ..., 120, 119, 118]], dtype=uint8)
 
-From the output above, we can see that every value in ``img[:,:,0]`` is an
-integer value between 0 and 255, representing the level of red in each
-corresponding image pixel (keep in mind that this might be different if you
-use your own image instead of `scipy.misc.face`).
+Every value is an integer from 0 to 255, representing the level of red in each
+of the 768 * 1024 = 786,432 red pixels.
 
 As expected, this is a 768x1024 matrix::
 
     >>> img[:, :, 0].shape
     (768, 1024)
 
-Since we are going to perform linear algebra operations on this data, it might
-be more interesting to have real numbers between 0 and 1 in each entry of the
-matrices to represent the RGB values. We can do that by setting
+To perform linear algebra, let's rescale these values into real numbers
+between 0 and 1:
 
     >>> img_array = img / 255
 
-This operation, dividing an array by a scalar, works because of NumPy's
-:ref:`broadcasting rules <array-broadcasting-in-numpy>`). (Note that in
-real-world applications, it would be better to use, for example, the
-:func:`img_as_float <skimage.img_as_float>` utility function from
-``scikit-image``).
-
-You can check that the above works by doing some tests; for example, inquiring
-about maximum and minimum values for this array::
+You can check that the rescaling worked by inquiring about maximum and minimum values::
 
     >>> img_array.max(), img_array.min()
     (1.0, 0.0)
 
-or checking the type of data in the array::
+or checking the type::
 
     >>> img_array.dtype
     dtype('float64')
 
-Note that we can assign each color channel to a separate matrix using the slice
-syntax::
+Using slice syntax, we can assign the channels to separate matrices::
 
     >>> red_array = img_array[:, :, 0]
     >>> green_array = img_array[:, :, 1]
     >>> blue_array = img_array[:, :, 2]
 
 **Operations on an axis**
 
-It is possible to use methods from linear algebra to approximate an existing set
-of data. Here, we will use the `SVD (Singular Value Decomposition)
-<https://en.wikipedia.org/wiki/Singular_value_decomposition>`_ to try to rebuild
-an image that uses less singular value information than the original one, while
-still retaining some of its features.
+Methods from linear algebra can reduce data into a smaller approximate version
+that retains the most important features of the original. We'll do that using
+`SVD (Singular Value Decomposition)
+<https://blog.statsbot.co/singular-value-decomposition-tutorial-52c695315254?gi=2729cca75ba3>`_.
```

Comment:
As I mentioned in #16244, I would propose keeping the wikipedia link for maintainability, even at the expense of clarity in this case.

Reply:
The impermanence is a good point. I tried to get around this by saving the page in archive.org, but it didn't save the LaTeX. I made a screenshot of the article; we can link to that (giving credit, of course).  The Wikipedia article is so impenetrable to nonspecialists that it's like somebody's playing a joke.  If that's our only option I think we're better off leaving off the link.

Reply:
I've found a nice explanation that's already on archive.org: https://web.archive.org/web/20200102142016/https://www.cs.princeton.edu/picasso/mats/PCA-Tutorial-Intuition_jp.pdf.

Reply:
When we write a NumPy tutorial on writing NumPy tutorials, we should include @melissawm's point about choosing outside links carefully.